Platform/AlderlakeBoardPkg/AcpiTables/AcpiTables.inf

```diff
@@ -62,6 +62,7 @@
   Rtct/Rtct.act
   Lpit/Lpit.act
   Bert/Bert.aslc
+  Dptf/Dptf.asl
```

Comment:
When we add any asl file here, they get loaded up even if Feature flag is disabled in configuration.

Reply:
You can also prevent a table from being installed by returning an error status from PlatformUpdateAcpiTable(), but if it's a large table, it's good to avoid building it in.